css/email-signup-mobile.css

```
@@ -0,0 +1,269 @@
+  }
+}
+
+/* ==========================================================================
+   EXTRA SMALL DEVICES (480px and below)
+   ========================================================================== */
+
+@media (max-width: 480px) {
+  /* Tighter padding for very small screens */
+  .early-bird-signup {
+    padding: var(--space-md) var(--space-sm) !important;
+    margin: var(--space-md) -var(--space-sm) !important; /* Full-width effect */
```

Biome (2.1.2)

[error] Unexpected unknown function: -var

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)